src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue20131.cs

```diff
@@ -0,0 +1,27 @@
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue20131 : _IssuesUITest
+{
+	public Issue20131(TestDevice testDevice) : base(testDevice)
+	{
+	}
+
+	public override string Issue => "Shell Flyout Header and Footer not respecting safe area on iOS in landscape mode";
+
+	[Test]
+	[Category(UITestCategories.Shell)]
+	public void FlyoutShouldRespectSafeAreaInLandscape()
+	{
+		App.WaitForElement("Label");
+#if ANDROID || IOS
+		App.SetOrientationLandscape();
+#elif MACCATALYST
+		App.EnterFullScreen();
+#endif
+		VerifyScreenshot();
```

Comment:
Pending snapshots, already available in the latest build.

Example, Android:
![image](https://github.com/user-attachments/assets/1b6acd1f-590e-4839-b488-48c44f47ac40)

Could you commit the images?